### PR TITLE
Unified this action with map and detect lang actions

### PR DESCRIPTION
**Pull request by jleven**
There is a tour which identifies the 4 code changes.

1. Moved `codesee-map-action` and `codesee-detect-languages-action` into subfolders of this repo `map` and `detect-lang` respectively
2. Modified action.yml to call the sub actions using `run` and passing in the arguments with `env`
3. Modified the detect lang code to use `process.env` to get arguments
4. Deleted ruby (which we don't support yet), and upgraded one package to stop deprecation warnings.

### CODE REVIEW

File: map/src/action.js [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+  }
+}
+
+function getBooleanInput(paramName) {
+  return ["true", "True", "TRUE"].includes(process.env[paramName])
+    ? true
+    : false;
+}
+
+function getConfig() {
```
Comment (jleven) [inline map/src/action.js:98]: This entire function was changed to use `process.env` instead of `core.getInput`

File: action.yml [hunk 2]
```diff
@@ -31,7 +31,8 @@ runs:
 
     - name: Detect Languages
       id: detect-languages
-      uses: Codesee-io/codesee-detect-languages-action@latest
+      run: node ${{ github.action_path }}/detect-lang/dist/index.js
+      shell: bash
```
Comment (jleven) [inline action.yml:35]: This and 3 other similar changes in this file are the only other functional code change

File: action.yml [hunk 3]
```diff
@@ -49,12 +50,6 @@ runs:
         python-version: "3.x"
         architecture: "x64"
 
-    - name: Configure Ruby '3.x'
-      uses: ruby/setup-ruby@v1
-      if: ${{ fromJSON(steps.detect-languages.outputs.languages).ruby }}
-      with:
-        ruby-version: "3.0"
```
Comment (jleven) [inline action.yml:56]: I deleted ruby since we don't do ruby

File: detect-lang/package.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "codesee-detect-languages-action",
+  "version": "0.1.0",
+  "main": "dist/index.js",
+  "license": "Unlicensed",
+  "dependencies": {
+    "@actions/core": "^1.10.0",
```
Comment (jleven) [inline detect-lang/package.json:7]: Finally, I upgraded this from 1.5 to 1.10 to make the deprecation warnings go away

**Review by LizardEatsFlies — APPROVED** [review]
good to 🐐

**Pull request merged**